cpp/zimt/compare.cc

```diff
@@ -246,7 +246,7 @@ int Main(int argc, char* argv[]) {
       for (size_t channel_index = 0; channel_index < file_b->Info().channels;
            ++channel_index) {
         const EnergyAndMaxAbsAmplitude new_energy_and_max_abs_amplitude =
-            NormalizeAmplitude(file_a_max_abs_amplitude,
```

Comment:
Nice, but now you can remove `file_a_max_abs_amplitude` completely, right?

Reply:
Yes, I do think so. Good catch.

Reply:
Great!

If you handle https://opensource.google/documentation/reference/cla, I can patch this PR to remove the variable completely if you don't want to do it yourself.

Reply:
I tried to patch the PR myself, and I think it worked. Learning a lot here :)